docs/faq.rst

```diff
@@ -306,3 +306,45 @@ Access full logs
 
 If you want to access the full logs you need to write ``-q`` and ``-v`` as
 individual tox arguments and avoid combining them into a single one.
+
+Running within a Docker container
+---------------------------------
+
+If you want to run tox within a Docker container you can use `31z4/tox <https://hub.docker.com/r/31z4/tox>`_.
+This Docker image neatly packages tox along with common build dependencies (e.g., ``make``, ``gcc``, etc) and currently
+`active CPython versions <https://devguide.python.org/versions/#status-of-python-versions>`_. See more details in
+its `GitHub repository <https://github.com/31z4/tox-docker>`_.
+
+The recommended way of using the image is to mount the directory that contains your tox configuration files and your
+code as a volume. Assuming your project is within the current directory of the host, use the following command to run
+tox without any flags:
+
+.. code-block:: shell
+
+    docker run -v `pwd`:/home/tox/tests -it --rm 31z4/tox
+
+Because an entry point of the image is ``tox``, you can easily pass subcommands and flags:
+
+.. code-block:: shell
+
+    docker run -v `pwd`:/home/tox/tests -it --rm 31z4/tox run-parallel -e black,py311
```

Comment:
Would prefer to mount the project under the /w folder, personal preference 😅

Reply:
That's possible too ☺️ But you have to specify the workdir in that case (`-w /w`):

    docker run -v `pwd`:/w -w /w -it --rm 31z4/tox run-parallel -e black,py311

I can change image workdir from `/home/tox/tests` to something more concise, like `/tests` maybe. I agree that `/home/tox/tests` might be too long to type 😅

Reply:
I'd make/w the default, concise better 😅

Reply:
@31z4 ping on this?

Reply:
I've been pretty busy recently. Hopefully will get a chance to work on the official image later this month. In the meantime https://github.com/31z4/tox-docker should be good enough for production already.